posthog/api/test/test_decide.py

```diff
@@ -134,14 +135,14 @@ def test_feature_flags(self):
             created_by=self.user,
         )
 
-        with self.assertNumQueries(4):
+        with self.assertNumQueries(5):
```

Comment:
Tangentially related, but probably worth documenting the expected queries? Otherwise this change is inconsequential.